project-management/src/main/java/life/qbic/projectmanagement/application/sample/SampleRegistrationServiceV2.java

```diff
@@ -89,10 +89,8 @@ public CompletableFuture<Void> registerSamples(Collection<SampleRegistrationInfo
         request.analyte(),
         request.analysisMethod(),
         request.comment(),
-        request.confoundingVariables(),
-        request.experimentId(),
-        request.projectId()
-    )).toList();
+        request.confoundingVariables(), projectId.value(), experimentReference.id())
```

Comment:
The order or arguments changed here. Is this intentional? experiment and project identifiers flipped.

Reply:
that is what happens when dropping usage of contextual objects :D